### PR TITLE
Lint object curly spacing

### PR DESCRIPTION
**Pull request by tyom**
So we never have to mention it in PRs again.

### CODE REVIEW

**Comment by teneightfive** [comment]
👍

File: webpack.config.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ module.exports = {
     new webpack.DefinePlugin({
       'process.env': {
         'NODE_ENV': JSON.stringify('production')
-      }}),
+      } }),
```
Comment (teneightfive) [inline webpack.config.js]: This one actually feels like it should be on the next line down.

Comment (tyom) [inline webpack.config.js]: Just that one? It's auto`--fix`ed. But I'll fixed it for you. ✨

Comment (teneightfive) [inline webpack.config.js]: The linter probably wouldn't spot that though. Just the lack of space.

It should be this though:
```
new webpack.DefinePlugin({
  'process.env': {
    'NODE_ENV': JSON.stringify('production')
  }
}),
```

Comment (tyom) [inline webpack.config.js]: Thanks Dom ✋

**Pull request merged**